src/main/java/org/openlmis/stockmanagement/controller/StockCardController.java

```diff
@@ -108,6 +145,64 @@ public ResponseEntity getStockCards(@PathVariable Long facilityId,
         }
     }
 
+    @RequestMapping(value = "facilities/{facilityId}/stockCards/adjust", method = POST, headers = ACCEPT_JSON)
```

Comment:
I assume this means we are not requiring our APIs to be RESTful? I don't think REST APIs typically have verbs in the URL?